common/src/main/java/com/wynntils/core/notifications/MessageContainer.java

```diff
@@ -9,17 +9,25 @@
 
 public class MessageContainer {
     protected TextRenderTask message;
+    private int iterations;
 
     public MessageContainer(String message) {
         this.message = new TextRenderTask(message, TextRenderSetting.DEFAULT);
+        this.iterations = 0;
```

Comment:
shouldn't we start this from 1?

So, when we have a message like this again, we have 2 "iterations". Maybe we could rename the variable to `duplicateCount` so it is clearer?

Reply:
Feels a bit weird to have it start from 1 imo

Reply:
yeah i come from a language where arrays start at 1 (lol it's fucked up) so i thought that way in the initial implementation of this PR but it's not hard to adjust the logic to have it make sense this way